amp/WEB-INF/src/org/digijava/module/aim/util/ActivityVersionUtil.java

```diff
@@ -346,6 +346,16 @@ public static AmpActivityVersion initializeActivity(AmpActivityVersion act) thro
         return act;
     }
     
+    public static Map<Long, Map<String, List<CompareOutput>>> compareActivities(Long[] ActivitiesId) throws Exception {
+        
+        Session session = PersistenceManager.getCurrentSession();
+        Map<Long, Map<String, List<CompareOutput>>> activitiesCollection = new HashMap<Long, Map<String, List<CompareOutput>>>();
```

Comment:
For further development its good you used a map here

Reply:
what do you mean?

Reply:
That using a map of longs instead of a list is good choice